src/components/Footer/Footer.module.scss

```diff
@@ -0,0 +1,25 @@
+@import 'styles/uswdsDependencies';
+
+.footer {
+  background: #333;
+  color: #fff;
+  @include u-font-size('body', '2xs');
```

Comment:
I never know when to use '2xs' or something similar with this utility and when to use another value, e.g. integer. How much does it matter?

Reply:
so usually I'll see what the `px` value is supposed to be in the design, and then look up the corresponding closest value in USWDS. these predefined values are referred to as [design tokens](https://designsystem.digital.gov/design-tokens/) and in this case there's a list of all the font size values at https://designsystem.digital.gov/design-tokens/typesetting/font-size/

Reply:
wonderful! thank you! i've been doing something similar but think i was looking at system tokens rather than theme tokens.